release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1220,6 +1236,10 @@ This update corrects the accepted label specification so that the plug-in import
 
 * Previously, the cluster ingress router was blocked from working because the Ingress Operator failed to configure a wildcard DNS record for the cluster ingress router on Azure Stack Hub IPI. With this fix, the Ingress Operator now uses the configured ARM endpoint to configure DNS on Azure Stack Hub IPI. As a result, the cluster ingress router now works properly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2032566[*BZ#2032566*])
 
+* Previously, the cluster-wide prox configuration could not accept IPv6 addresses for the `noProxy` setting. Consequently, it was impossible to install a cluster whose configuration was having `noProxy` with IPv6 addresses. With this update, the Cluster Network Operator is now able to parse IPv6 addresses for the `noProxy` setting of the cluster-wide proxy resource. As a result, it is now possible to exclude IPv6 addresses from proxying. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939435[*BZ#1939435*])
+
+* Before {product-title} 4.8, the IngressController API did not have any subfields under the `status.endpointPublishingStrategy.hostNetwork` and `status.endpointPublishingStrategy.nodePort` fields. These fields could be null even if the `spec.endpointPublishingStrategy.type` was set to `HostNetwork` or `NodePortService`. In {product-title} 4.8, the `status.endpointPublishingStrategy.hostNetwork.protocol` and `status.endpointPublishingStrategy.nodePort.protocol` subfields were added, and the Ingress Operator set default values for these subfields when the Operator admitted or re-admitted an IngressController that specified the "HostNetwork" or "NodePortService" strategy type. With this bug, however, the Operator ignored updates to these spec fields, and updating `spec.endpointPublishingStrategy.hostNetwork.protocol` or `spec.endpointPublishingStrategy.nodePort.protocol` to `PROXY` to enable PROXY protocol on an existing IngressController had no effect. To work around this issue, it was necessary to delete and recreate the IngressController to enable PROXY protocol. With this update, the Ingress Operator is changed so that it correctly updates the status fields when `status.endpointPublishingStrategy.hostNetwork` and `status.endpointPublishingStrategy.nodePort` are null and when the IngressController spec fields specify PROXY protocol with the `HostNetwork` or `NodePortService` endpoint publishing strategy type. As a result, setting `spec.endpointPublishingStrategy.hostNetwork.protocol` or `spec.endpointPublishingStrategy.nodePort.protocol` to `PROXY`now takes proper effect on upgraded clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1997226[*BZ#1997226*])
```

Comment:
you have PROXY in backticks for some and some are not. Are they all supposed to have backticks or not have backticks?

Reply:
I think they're just context dependent. Brandi wrote this one; but for example, when you set something to `proxy`, you would put it in backticks. When it's proxy protocol I guess it's a noun?